config/config.toml

```diff
@@ -147,4 +147,9 @@ peer_seeds = [
 committee = ["2ea0df3059caf4411624d6bf5b9c02238d607d2798c586b3e6c2a054da3f205a"] # cannot be of zero size
 phase_timeout = 30
 template_id = "EditableMetadata"
-
+[collectibles]
```

Comment:
**Q:** Why do we have this duplicate configuration options, when we have the master config files in `common/config/presets`? I thought we did away with duplicate configs.

Reply:
Honestly, I don't know. But since the config is there, I added the collectibles part.

Reply:
Ok, Mike explained, the file is deleted in this PR.